README.md

```diff
@@ -33,13 +33,21 @@ appveyor_agent '3.12.0' do
 end
 ```
 
-### Deploy
-`appveyor_deploy` start the deployment for the specified environment in AppVeyor
+### Start Deployment
+The `appveyorapi_deploy` LWRP can be used to start the deployment for the specified environment in Appveyor CI using its API.
+
 ```ruby
```

Comment:
Why are we switching the build version out of the top and switching it for environment name?